### PR TITLE
config.md: minor changes for process

### PR DESCRIPTION
**Pull request by Mashimiao**
Signed-off-by: Ma Shimiao <mashimiao.fnst@cn.fujitsu.com>

### CODE REVIEW

File: config.md [hunk 1]
```diff
@@ -132,14 +132,14 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
+* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the processes in the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
```
Comment (wking) [inline config.md]: They may not all have these caps anyway (e.g. you could inject in a new process into the container with more caps, or drop the caps in the main process or one of its children).  This is just setting the caps for the process configured by `process`, so we should probably follow  the earlier properties in this section and replace “the process(es) inside the container” with “the process”.

File: config.md [hunk 2]
```diff
@@ -119,11 +119,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
 ## <a name="configProcess" />Process
 
-**`process`** (object, OPTIONAL) specifies the container process.
+**`process`** (object, OPTIONAL) specifies a process to run inside the container.
   This property is REQUIRED when [`start`](runtime.md#start) is called.
 
-* **`terminal`** (bool, OPTIONAL) specifies whether a terminal is attached to that process, defaults to false.
-  As an example, if set to true on Linux a pseudoterminal pair is allocated for the container process and the pseudoterminal slave is duplicated on the container process's [standard streams][stdin.3].
+* **`terminal`** (bool, OPTIONAL) specifies whether a terminal is attached to the process, defaults to false.
+  As an example, if set to true on Linux a pseudoterminal pair is allocated for the process and the pseudoterminal slave is duplicated on the process's [standard streams][stdin.3].
```
Comment (wking) [inline config.md]: While we touch this line, can we shift to four-space indents?  Or we can punt that to a PR that just does whitespace adjustment for `process` (like #724 is doing for hooks).

Comment (Mashimiao) [inline config.md]: Let's punt that to a new PR. There are too many lines starting with two-space indents

File: config.md (hunk 1 above)
Comment (Mashimiao) [inline config.md]: fixed, thanks

File: config.md [hunk 3]
```diff
@@ -132,14 +132,14 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
 * **`args`** (array of strings, REQUIRED) with similar semantics to [IEEE Std 1003.1-2001 `execvp`'s *argv*][ieee-1003.1-2001-xsh-exec].
   This specification extends the IEEE standard in that at least one entry is REQUIRED, and that entry is used with the same semantics as `execvp`'s *file*.
-* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process(es) inside the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
+* **`capabilities`** (object, OPTIONAL) is an object containing arrays that specifies the sets of capabilities for the process in the container. Valid values are platform-specific. For example, valid values for Linux are defined in the [capabilities(7)][capabilities.7] man page, such as `CAP_CHOWN`. Any value which cannot be mapped to a relevant kernel interface MUST cause an error.
```
Comment (wking) [inline config.md]: I [still][1] think we want to drop “in(side) the container” here, like you're doing up above for `terminal`.  And we should do that down in `rlimits` too.  The initial `process` line establishes that the process is inside the container, and all these properties under `process` can just use “the process”.

[1]: https://github.com/opencontainers/runtime-spec/pull/809#discussion_r116297660

Comment (Mashimiao) [inline config.md]: OK, updated.

File: config.md [hunk 4]
```diff
@@ -148,20 +148,20 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
...
     If `rlimits` contains duplicated entries with same `type`, the runtime MUST error out.
 
-* **`noNewPrivileges`** (bool, OPTIONAL) setting `noNewPrivileges` to true prevents the processes in the container from gaining additional privileges.
+* **`noNewPrivileges`** (bool, OPTIONAL) setting `noNewPrivileges` to true prevents the process from gaining additional privileges.
   As an example, the ['no_new_privs'][no-new-privs] article in the kernel documentation has information on how this is achieved using a prctl system call on Linux.
 
 For Linux-based systems the process structure supports the following process-specific fields.
 
-* **`apparmorProfile`** (string, OPTIONAL) specifies the name of the AppArmor profile to be applied to processes in the container.
+* **`apparmorProfile`** (string, OPTIONAL) specifies the name of the AppArmor profile to be applied to process.
```
Comment (wking) [inline config.md]: “to process” → “to the process”.

Comment (Mashimiao) [inline config.md]: fixed, thanks

**Comment by Mashimiao** [comment]
@opencontainers/runtime-spec-maintainers  PTAL

File: config.md [hunk 5]
```diff
@@ -119,11 +119,11 @@ For Windows, see [mountvol][mountvol] and [SetVolumeMountPoint][set-volume-mount
 
 ## <a name="configProcess" />Process
 
-**`process`** (object, OPTIONAL) specifies the container process.
+**`process`** (object, OPTIONAL) specifies a process to run inside the container.
```
Comment (tianon) [inline config.md]: IMO "the container process" has a very different connotation here than simply "a process to run", especially in the context of a PID namespace (which can only exist as long as "the" container process does), so this change doesn't make a lot of sense to me.

Comment (Mashimiao) [inline config.md]: OK, updated

**Comment by crosbymichael** [comment]
@Mashimiao can you rebase?

**Comment by Mashimiao** [comment]
@opencontainers/runtime-spec-maintainers  rebased.  PTAL

File: config.md (hunk 4 above)
Comment (wking) [inline config.md]: > fixed, thanks

Still not fixed in 04864b7c.

Comment (wking) [inline config.md]: And maybe "to be applied to" -> "for", to get the same substance in fewer words.

Comment (Mashimiao) [inline config.md]: sorry, may left it behind when doing rebase. updated.

**Comment by tianon** [comment]
LGTM

**Comment by hqhq** [comment]
LGTM

**Pull request merged**